pkg/interfaces.go

```diff
@@ -22,19 +22,4 @@ type Caller interface {
 }
 
 // tokenExpiredCallback is the callback function to handle the case when the vault token has already expired
```

Comment:
comment is out of dated, update

Reply:
You  mean `tokenExpiredCallback ` => `TokenExpiredCallback` ?

Reply:
yea